stream-chat-android-ui-components-sample/src/main/kotlin/io/getstream/chat/ui/sample/feature/chat/ChatFragment.kt

```diff
@@ -56,6 +57,11 @@ class ChatFragment : Fragment() {
         configureBackButtonHandling()
     }
 
+    override fun onResume() {
+        super.onResume()
```

Comment:
Good, I was worried that client would use incorrect resize mode which I defined in the manifest 👍